index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <p>
+    Lorem ipsum dolor sit, amet consectetur adipisicing elit. Similique,
+    laudantium!
+  </p>
```

Comment:
I think it would be better to use semantic tags here.
With a semantic tag, you can easily differentiate your code part, and its a good coding practice

Reply:
Good job being suggestive and not forming your suggestion as an order.